### PR TITLE
WIP: Add code of conduct

### PR DESCRIPTION
**Pull request by HiddenBug**
This PR adds a default code of conduct to the community health files.

### CODE REVIEW

**Comment by HiddenBug** [comment]
It was mentioned in PR #1 in comment https://github.com/PilzDE/.github/pull/1#issuecomment-549716253 that we could/should choose the same code of conduct as the MoveIt-Project.

**Comment by HiddenBug** [comment]
> It was mentioned in PR #1 in comment [#1 (comment)](https://github.com/PilzDE/.github/pull/1#issuecomment-549716253) that we could/should choose the same code of conduct as the MoveIt-Project.

I want to make some statements regarding this suggestion.
First off, even so I'm going to argue a little bit, I'm relatively open to any kind of suggestions and I'm not generally opposed to the MoveIt/Django/Speak Up! code of conduct. However, I also see some drawbacks:
- What disturbs me a little bit when it comes to the MoveIt/Django/Speak Up!  code of conduct: We are simply no community yet. Our development is mainly driven by us and the [Speak Up! Code of conduct](http://web.archive.org/web/20141109123859/http://speakup.io/coc.html) speaks more from a community point of view whereas the  [Contributor Covenant](https://www.contributor-covenant.org/) argues more from a project maintainer view.
- I also want to mention that the "Contributor Covenant" already was adopted by a wide range of projects like Linux, git, Atom, etc. for more see the [Contributor Covenant - Adopters](https://www.contributor-covenant.org/adopters)
- The fact that is widely used also should be helpful when it comes to improving the text or finding errors etc. We then simply have to update to a new version of the text.

What do you think? @agutenkunst @ct2034 @leunMar @dbakovic @jschleicher

**Comment by jschleicher** [comment]
Please follow the discussion in those moveit threads, especially https://github.com/ros-planning/moveit/pull/1664#issuecomment-535597922 about "policemen". I like the short-and-precise django texts better than the overwhelming text from contributor covenant.

From a practical perspective: I agree that this project(s) are mostly "inside Pilz"; but then the "enforcement" section is pointless. If I write a mail to "ros@pilz", who exactly is responsible to mediate? Thus I suggest to stick to the values and standards as in [the moveit coc](https://github.com/ros-planning/moveit/blob/master/CODE_OF_CONDUCT.md).

**Comment by HiddenBug** [comment]
Other opinions?  @agutenkunst @ct2034 @leunMar @dbakovic

**Comment by leunMar** [comment]
I lean towards the moveit COC. Imho it is important which language is chosen in such a document.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ However, if you think that something is missing or if you think that something
 is incorrectly or badly explained, feel free to open a pull request to improve
 this document.
 
+## Code of conduct
+We expect everyone who wants to participate in our projects to uphold our
```
Comment (leunMar) [inline CONTRIBUTING.md]:
```suggestion
We maintain a code of conduct that we would like to promote to all contributors:
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+This Code of Conduct applies within all project spaces, and it also applies when
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
```
Comment (ct2034) [inline CODE_OF_CONDUCT.md]: this is a good method of enforcement

**Comment by ct2034** [comment]
I like the moveit coc. I think it works for our purpose, too. 
@jschleicher I am also not in favour of beeing policemen but I think these concerns by @gavanderhoorn are taken into account in the version of coc, we refer to.

**Review by ct2034 — APPROVED** [review]

**Comment by rfeistenauer** [comment]
I would agree to @jschleicher and @leunMar. The moveit CoC is shorter, friendlier and nicer to read (and yes i deem that a criteria). It states the core points short and more pregnant. 
We can add the ros@pilz mail, but if someone feels for whatever reason harassed by one of us, how should he know he is not addressing exactly that person by writing to that address.

**Comment by rfeistenauer** [comment]
We will take the moveit CoC

**Pull request closed without merging**